docs_dev/releasing.md

```diff
@@ -179,27 +185,47 @@ Alternatively you can use the UI.
    ```
    * Set the tag to be the correct version for the tag.
 
-## Version the website
+# Version the website
 
 The main Kubeflow website at [www.kubeflow.org](www.kubeflow.org) points to the
-**master** branch of the `kubeflow/website` repo. Similarly, 
-[master.kubeflow.org](https://master.kubeflow.org/) also points to the master
-branch.
+version of the website we want users to see.
+
+* Most of the time this will correspond to the **master** branch of the `kubeflow/website` repo
+
+* However, leading up to a minor release (0.Y) [www.kubeflow.org](www.kubeflow.org) 
```

Comment:
Should we change `0.Y` to `vX.Y`? (Because soon the `0` will not mean much, as we've moving to v1.) Also, `vX.Y` is consistent with the section "Creating and publishing a website branch for vX.Y".)